.github/component_owners.yml

```diff
@@ -58,3 +58,6 @@ components:
     - trask
   static-instrumenter:
     - anosek-an
+  kafka-exporter:
+    - spockz
+    - vincentfree
```

Comment:
@naser-ayat did you want to be a component owner as well?

Reply:
Two component owners should be enough. Thanks for the suggestion.

Reply:
@vincentfree Could you please approve the PR?

Reply:
it would be great to get approvals from both of the component owners

Reply:
Approved on my part 👍

Reply:
@vincentfree can you approve via github? thx!

![image](https://github.com/open-telemetry/opentelemetry-java-contrib/assets/218610/8f32c0ec-5732-43dc-a060-e54dbbafbb8f)

Reply:
@spockz ^^ can you approve via github also?

Reply:
Done, sorry was going to do an actual review but I did that before so that's fine now. finally came around to click that button :)